openapi.yaml

```diff
@@ -44,4 +44,7 @@ paths:
   # report endpoint
   /reports:
     $ref: "./paths/report.yaml#/Report"
-    
\ No newline at end of file
+  
+  # ml seller endpoint
+  /predict:
```

Comment:
Non mandatory. You can use any endpoint you want because the server for the ML is different with the backend service.

Reply:
Aight, I've already remove it. But, in the FastAPI services, the endpoint is still same at `/predict`.